src/bitcoind/mod.rs

```diff
@@ -252,6 +314,16 @@ pub fn bitcoind_main_loop(
                         ))
                     })?;
             }
+            BitcoindMessageOut::CPFPTransaction(txids, feerate, resp_tx) => {
+                log::trace!("Received 'cpfptransaction' from main thread");
+
+                resp_tx
+                    .send(cpfp(revaultd, bitcoind, txids, feerate))
+                    .map_err(|e| {
+                        BitcoindError::Custom(format!("Sending transaction for CPFP: {}", e))
+                    })?;
+                return Ok(());
```

Comment:
Heh, you copy pasted this from the ShutDown handler i guess.. But that's going to stop the bitcoind thread once you have CPFPed once. All other calls will make the daemon crash, and actually this will make it crash when stopping it as well...